content/docs/addons-test-utils.md

```diff
@@ -137,11 +137,11 @@ mockComponent(
 )
 ```
 
-Pass a mocked component module to this method to augment it with useful methods that allow it to be used as a dummy React component. Instead of rendering as usual, the component will become a simple `<div>` (or other tag if `mockTagName` is provided) containing any provided children.
+इस मेथड के लिए एक नकली कौम्पोनॅन्ट मॉड्यूल को पारित करें इसे उपयोगी मेथडयों के साथ संवर्धित करें जो इसे डमी React कौम्पोनॅन्ट के रूप में उपयोग करने की अनुमति देता है। हमेशा की तरह रेंडर करने के बजाय, कौम्पोनॅन्ट एक सरल `<div>` (या अन्य टैग यदि `mockTagName` दिया गया है) बन जाएगा जिसके अंदर दिए गए चिल्ड्रन होंगे।
```

Comment:
नकली is fake which is different

Reply:
हमेशा की तरह रेंडर करने could be worded better

Reply:
इस मेथड के लिए एक **प्रतिरूप** कौम्पोनॅन्ट मॉड्यूल को पारित करें इसे उपयोगी मेथडयों के साथ संवर्धित करें जो इसे डमी React कौम्पोनॅन्ट के रूप में उपयोग करने की अनुमति देता है। **सामान्य रूप से रेंडर करने के बजाय**, कौम्पोनॅन्ट एक सरल `<div>` (या अन्य टैग यदि `mockTagName` दिया गया है) बन जाएगा जिसके अंदर दिए गए चिल्ड्रन होंगे।